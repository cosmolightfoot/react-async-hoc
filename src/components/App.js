import React from 'react';
import ErrorBoundaries from './ErrorBoundaries';
import Bad from './Bad';
import Good from './Good';

export default function App() {
  return (
    <>
      <ErrorBoundaries>
        <Bad />
      </ErrorBoundaries>
      <Good />
    </>
  );
}
