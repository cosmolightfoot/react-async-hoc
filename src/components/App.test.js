import React from 'react';
import { shallow } from 'enzyme';
import App from './App.js';

describe('Error Boundary tests', () => {
  it('renders the app component', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});



