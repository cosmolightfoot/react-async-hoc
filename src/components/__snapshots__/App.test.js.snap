// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Boundary tests renders the app component 1`] = `
<Fragment>
  <ErrorBoundaries>
    <Bad />
  </ErrorBoundaries>
  <Good />
</Fragment>
`;
